client/src/components/EditProfile/InputControls/TextFieldWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { TextField, TextFieldProps } from '@material-ui/core';
+import { FieldHookConfig, useField, useFormikContext } from 'formik';
+
+interface otherProps {
+  placeholder?: string;
+  multiline?: boolean;
+  rows?: number;
+  type?: string;
+  InputLabelProps?: { shrink: boolean };
+  select?: boolean;
+  variant?: string;
+}
+
+const TextFieldWrapper: React.FC<otherProps & TextFieldProps & FieldHookConfig<string>> = ({
```

Comment:
@ugur-akin May be you can take a look at this part of the PR, am not sure about this component